tests/suites/casper/formfill.js

```diff
@@ -1,5 +1,6 @@
 /*eslint strict:0*/
 var fs = require('fs');
+var x = require('casper').selectXPath;
 
```

Comment:
I thought that it is recommended pattern for XPath expressions according this example http://docs.casperjs.org/en/latest/selectors.html#index-2